tests/functional/modules/pyi_pkg_resources_provider/package/setup.py

```diff
@@ -15,19 +15,17 @@
 
 import setuptools
 
-setuptools.setup(
-    name='pyi_pkgres_testpkg',
-    version='1.0.0',
-    setup_requires="setuptools >= 40.0.0",
-    author='PyInstaller development team',
-    packages=setuptools.find_packages(),
-    package_data={
-        "pyi_pkgres_testpkg": [
-            "subpkg1/data/*.txt",
-            "subpkg1/data/*.md",
-            "subpkg1/data/*.rst",
-            "subpkg1/data/extra/*.json",
-            "subpkg3/*.json",
-        ],
-    }
-)
+setuptools.setup(name='pyi_pkgres_testpkg',
+                 version='1.0.0',
+                 setup_requires="setuptools >= 40.0.0",
+                 author='PyInstaller development team',
+                 packages=setuptools.find_packages(),
+                 package_data={
+                     "pyi_pkgres_testpkg": [
+                         "subpkg1/data/*.txt",
+                         "subpkg1/data/*.md",
+                         "subpkg1/data/*.rst",
+                         "subpkg1/data/extra/*.json",
+                         "subpkg3/*.json",
+                     ],
+                 })
```

Comment:
If you use a trailing comma at the end of any sequence
```suggestion
    },
)
```
then yapf will put it back to:
```
setup(
    foo,
    bar,
)
```

Reply:
Which I think looks better for huge function calls like that.

Reply:
Agreed.